src/charts/tableView.js

```diff
@@ -12,14 +12,16 @@ define(['jquery', 'moment', 'lokijs', 'charts/chartingRequestMap', 'websockets/s
     table.animate({left: '0'}, 250);
     chart.animate({left: '-100%'}, 250);
     refresh_table(dialog, key); /* clear table and fill it again */
+    dialog.parent().css('width', 520);
     dialog.view_table_visible = true; /* let stream_handler new ohlc or ticks update the table */
   }
   var hide_table_view = function(dialog){
     var table = dialog.find('.table-view');
     var chart = dialog.find('.chart-view');
     table.animate({left: '100%'}, 250);
-    chart.animate({left: '0'}, 250);
+    chart.animate({ left: '0' }, 250);
     dialog.view_table_visible = false;
+    dialog.parent().css('width', 350);
```

Comment:
You should set the width to previous dialog size, the user might have resized the dialog.
Or it might be in fullscreen.